test.e2e/spec/services.spec.js

```diff
@@ -655,6 +655,26 @@ define(['angular', 'given', 'util'], function(angular, given, util) {
           });
       });
 
+      it('clear user upon failed auth on User.getCurrent', function(done) {
+        return givenLoggedInUser('to-be-deleted@example.com')
+          .then(function(loggedInUser) {
+            return Customer.delete({ id: loggedInUser.userId }).$promise;
+          })
+          .then(function(deleteResponse) {
+            return Customer.getCurrent().$promise;
+          })
+          .then(function(res) {
+            throw new Error('User.getCurrent() was supposed to fail.');
+          }
+          , function(res) {
```

Comment:
Nitpick: I find this formatting weird. We use one-arg-per-line rule in the rest of our codebase, see https://github.com/strongloop/loopback-contributor-docs/blob/45e4dfc18718bf21077fd43053e08464b111acf3/style-guide.md#one-arg-per-line.